common/azureml_appinsights_logger/README.md

```diff
@@ -0,0 +1,147 @@
+[![Build Status](https://dev.azure.com/cse-manufacturing/MLOpsManufacturing/_apis/build/status/observability-library/observability-lib-ci?branchName=refs%2Fpull%2F59%2Fmerge)](https://dev.azure.com/cse-manufacturing/MLOpsManufacturing/_build/latest?definitionId=44&branchName=refs%2Fpull%2F59%2Fmerge)
+[![Build Status](https://dev.azure.com/cse-manufacturing/MLOpsManufacturing/_apis/build/status/observability-library/observability-lib-cd?branchName=paige%2Fobservability_lib)](https://dev.azure.com/cse-manufacturing/MLOpsManufacturing/_build/latest?definitionId=45&branchName=paige%2Fobservability_lib)
+
+# Overview
+Logs text and metrics in Azure ML, App Insights, and console with a single call. Also supports tracing in App Insights. 
+
+> Disclaimer: This package is built by Microsoft engineers, but it is not an officially supported Microsoft package.
+
+# How to use
+
+## 1. Add dependencies
+
+The following dependencies are required to use this package.
+
+```yaml
+  - python-dotenv==0.15.*
+  - dataclasses==0.*
+  - opencensus==0.7.*
+  - opencensus-context==0.1.*
+  - opencensus-ext-azure==1.0.*
+  - opencensus-ext-logging==0.1.*
+  - opencensus-ext-httplib==0.7.*
+```
+ 
+## 2. Configure environment variables to control logger behavior
+
+```bash
+# AppInsights logger is enabled if this is a valid connection string
+APPLICATIONINSIGHTS_CONNECTION_STRING = ''
+# Console logger is enabled if set to 'true'
+LOG_TO_CONSOLE = 'false'
+# Azure ML logger is enabled when running as an experiment in Azure ML
+# DEBUG, INFO, WARNING, ERROR, CRITICAL, default is WARNING
+LOG_LEVEL = 'WARNING' 
+# Probability 0.0 -> 1.0
+LOG_SAMPLING_RATE = '1.0'
+# Probability 0.0 -> 1.0
+TRACE_SAMPLING_RATE = '1.0'
+# Frequency in seconds to send metric to App Insights, default is 15
+METRICS_EXPORT_INTERVAL = '15'
+# Whether to log App Insights standard machine metrics, CPU, memory etc.
+ENABLE_STANDARD_METRICS = 'true'
```

Comment:
Should the default value be false?
Are we going to capture the metrics from the machine that is running our script?
I found this polluting our logs in Application insights.

Reply:
Sure wouldn't mind if this is set to `false`. I found it useful looking and traversing through the Application Map that these insights are already populated by default.